CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+# Contributing
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+We welcome contributions via both the raising of issues and submitting pull requests. But before you
+do, please take a moment to consult the below.
+
+## Chatting
+
+If before any of the below you'd like to discuss an issue or some code, then come and chat with
+us on one of the following platforms:
```

Comment:
Confused at the meaning here:  "If before any of the below you'd like to discuss an issue or some code"

Reply:
Reworded.